### PR TITLE
docs(beginner-guide): Preparing the MCDU

### PR DESCRIPTION
**Pull request by Valastiri**
Will update this comment as the guide progresses

Sections Complete

- [x] Data
- [x] Init A
- [x] Flight Plan
- [x] Secondary Flight Plan
- [x] Radnav
- [x] Init B
- [x] Prog
- [x] Perf
- [x] Images added
- [x] Adding Fuel
- [x] simBrief section
---
Sample Site for Review

To preview the built version download the zip file below.

- Extract the zip file and open the site directory
- Find `index.html` and open it. 
- Navigate to the beginner guide on the top menu
- Use the overview page to go to the MCDU guide OR use the page navigation on the left. 

[Site Demo](https://github.com/flybywiresim/docs/files/6475997/mcdu-guidev2.zip) 5-14-2021

### CODE REVIEW

File: docs/beginner-guide/preparing-mcdu.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
+# Preparing the MCDU
+
+This guide will help you prepare the MCDU in the A32NX for your departure. It includes a simple route that you can use to follow along easily and replicate in the simulator.
+
+The simBrief route used in this guide - [Available Here](../assets/beginner-guide/mcdu/sample-ofp.pdf)
```
Comment (scoopscode) [inline docs/beginner-guide/preparing-mcdu.md:5]: Isnt S in SimBrief supposed to be capitalized?

File: docs/beginner-guide/preparing-mcdu.md [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+    * Using the keypad type in `3.1 and press LSK2R
+    * You should load this amount of fuel via the EFB or AOC option.
+
+    ![mcdu16](../assets/beginner-guide/mcdu/mcdu16.png)
+
+    [Top of Section 3](#section-3)
+
+=== "PROG"
+
+    On this page you can see details about the flight data pertaining to the current phase oof flight.
```
Comment (ExampleWasTaken) [inline docs/beginner-guide/preparing-mcdu.md]: You might wanna change "oof" to "of" ;)

**Comment by ExampleWasTaken** [comment]
I have two additional suggestions:
1. Note, that fuel and payload have to be entered before ZFW/CG for the end result to be correct.
2. Note, that waypoints, STARs, SIDs, etc. maybe called differently due to different nav-databases or different AIRAC cycles between SimBrief and the simulator.

**Comment by Valastiri** [comment]
@ExampleWasTaken

> 1. Note, that fuel and payload have to be entered before ZFW/CG for the end result to be correct.

Added a warning message to the ZFW/CG section
image


> 2. Note, that waypoints, STARs, SIDs, etc. maybe called differently due to different nav-databases or different AIRAC cycles between SimBrief and the simulator.

Do you have a location on where to note this possibly?

File: docs/beginner-guide/preparing-mcdu.md (hunk 1 above)
Comment (Valastiri) [reply docs/beginner-guide/preparing-mcdu.md:5]: Based on Navigraph's branding the "S" isn't capitalized unless start of sentence / title. Adjusted copy to reflect that.

File: docs/beginner-guide/preparing-mcdu.md [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+=== "FLIGHT PLAN"
+
+    Upon loading the flight plan page you should see three entries. Departure airport, `(DECEL)`, and arrival airport. 
+
+    Our route for this flight can be found on the 2nd page of the OFP
+
+    ![ofp3](../assets/beginner-guide/mcdu/ofp3.png)
+
+    `EGFF/30 BCN1A BCN N864 MONTY MIRSI1A EGCC/05R`
+
```
Comment (ExampleWasTaken) [inline docs/beginner-guide/preparing-mcdu.md:141]: @Valastiri With regard to my last point, I would maybe put an info field here.

Comment (ExampleWasTaken) [reply docs/beginner-guide/preparing-mcdu.md:141]: Well GitHub doesn't show the line, duh. I would put it in line 138.

**Comment by Valastiri** [comment]
Reviewed by Tarek on Discord. 

Added suggestions from other contributors. Updated original comment with latest build.

**Pull request merged**